### DISCUSSION TITLE
Advanced Select in laravel

### DISCUSSION THREAD

**Discussion by csimeon2023** [htmlstreamofficial/preline#167] [General]
I have tried the Advanced Select in laravel. I am having a hard time trying to figure how to pass the selected options to the controller, without having to use AJAX. Is there a way to add the "name" attribute to the Advanced Select?

**Comment by bsdrazor** [comment]
+ 1 question

What about optgroup and it's rendering ?

**Reply by csimeon2023** [reply]
It is rendering

**Reply by sureMOISE** [reply]
No, it isn't...the optgroup option is not rendered and any option inside it will be ignored

**Comment by csimeon2023** [comment]
I will give optgroup a try. However if I put <select name="anyfield" and dd(all) at controller level, the field is not listed nor recognized by the controller while other input fields in the same blade form are
